docs/resources/cdn_originshielding.md

```diff
@@ -0,0 +1,96 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "gcore_cdn_originshielding Resource - terraform-provider-gcore"
+subcategory: ""
+description: |-
+  Represent origin shielding
+---
+
+# gcore_cdn_originshielding (Resource)
+
+Represent origin shielding
+
+## Example Usage
+
+```terraform
+provider gcore {
+  permanent_api_token = "251$d3361.............1b35f26d8"
+}
+
+resource "gcore_cdn_origingroup" "origin_group_1" {
+  name     = "origin_group_1"
+  use_next = true
+  origin {
+    source  = "example.com"
+    enabled = true
+  }
+}
+
+resource "gcore_cdn_resource" "cdn_example_com" {
+  cname               = "cdn.example.com"
+  origin_group        = gcore_cdn_origingroup.origin_group_1.id
+  origin_protocol     = "MATCH"
+  secondary_hostnames = ["cdn2.example.com"]
+
+  options {
+    edge_cache_settings {
+      default = "8d"
+    }
+    browser_cache_settings {
+      value = "1d"
+    }
+    redirect_http_to_https {
+      value = true
+    }
+    request_limiter {
+      rate_unit = "r/s"
+      rate = 5
+      burst = 1
+    }
+    gzip_on {
+      value = true
+    }
+    cors {
+      value = [
+        "*"
+      ]
+    }
+    rewrite {
+      body = "/(.*) /$1"
+    }
+    webp {
+      jpg_quality = 55
+      png_quality = 66
+    }
+
+    tls_versions {
+      enabled = true
+      value = [
+        "TLSv1.2",
+      ]
+    }
+
+    force_return {
+      code = 200
+      body = "OK"
+    }
+  }
+}
+
+resource "gcore_cdn_originshielding" "origin_shielding_1" {
+  resource_id   = gcore_cdn_resource.cdn_example_com.id
+  shielding_pop = 1
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `resource_id` (Number) ID of CDN resource for which shielding will be applied
+- `shielding_pop` (Number) ID of the shielding point of present
```

Comment:
Maybe it makes a sense to add some instructions how to obtain the shielding_pop id, what do you think?

Reply:
Good point, i'll update examples and docs